R/slide.R

```diff
@@ -731,6 +788,134 @@ epi_slide_mean <- function(x, col_names, ..., before, after, ref_time_values,
   return(result)
 }
 
+#' Optimized slide function for performing rolling averages on an `epi_df` object
```

Comment:
The diff is grabbing things in a weird order; I'm guessing this is just the previous docs for `epi_slide_mean`?

Reply:
actually, the man page suggests something different. I'm a bit confused

Reply:
Diff and function documentation look fine to me. I put most of the args documentation in `epi_slide_opt` and inherited those fields here and for `epi_slide_sum` using the `inherit` and `inheritParams` roxygen tags. So the man pages look quite similar between `epi_slide_opt`, `epi_slide_mean`, and `epi_slide_sum`.